wpilibc/src/main/native/include/frc/apriltag/AprilTagFieldLayout.h

```diff
@@ -48,7 +52,9 @@ class WPILIB_DLLEXPORT AprilTagFieldLayout {
    *
    * @param apriltags Vector of AprilTags.
    */
-  explicit AprilTagFieldLayout(std::vector<AprilTag> apriltags);
+  explicit AprilTagFieldLayout(std::vector<AprilTag> apriltags,
```

Comment:
The explicit keyword is no longer needed because the constructor has more than one argument.